docs/source/models/gbd2017/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -28,50 +28,54 @@ Concept Model Diagram
 
+Time
+++++
+
+* Start and end year: **2020 -- 2025**
+* Simulation time step: **1 day** to capture short timeframe of diarrheal
+  diseases and neonatal causes
+
 Demographics
 ++++++++++++
 
+* Locations: **Nigeria, India, Ethiopia**
+* Population: **Prospective open cohort of 0-5 year-olds**
+* Size of largest starting population: **100,000 simulants**
+* Youngest start-age and oldest end-age: **0 -- 5 years**
+* Exit age (at what age to stop tracking simulants): **5 years**
+* Fertility: **Crude birth rate**
+
 GBD Causes
 ++++++++++
 
-:ref:`Measles <2017_cause_measles>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Measles <2017_cause_measles>`
 
-:ref:`Diarrheal Diseases <2017_cause_diarrhea>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Diarrheal Diseases <2017_cause_diarrhea>`
 
-:ref:`Neural Tube Defects <2017_cause_neural_tube_defects>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Neural Tube Defects <2017_cause_neural_tube_defects>`
 
-:ref:`Lower Respiratory Infections <2017_cause_lower_respiratory_infections>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Lower Respiratory Infections <2017_cause_lower_respiratory_infections>`
 
-:ref:`Neonatal Encephalopathy <2017_cause_neonatal_encephalopathy>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Neonatal Encephalopathy <2017_cause_neonatal_encephalopathy>`
 
-:ref:`Neonatal Sepsis <2017_cause_neonatal_sepsis>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Neonatal Sepsis <2017_cause_neonatal_sepsis>`
 
-:ref:`Neonatal Jaundice <2017_cause_neonatal_jaundice>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Neonatal Jaundice <2017_cause_neonatal_jaundice>`
 
-:ref:`Other Neonatal Disorders <2017_cause_neonatal_other>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Other Neonatal Disorders <2017_cause_neonatal_other>`
+  (Probably omit this cause, as we don't seem to have enough data to model it)
 
-PAF-of-1 Cause/Risk Pairs
-+++++++++++++++++++++++++
+GBD Risks
++++++++++
 
-:ref:`Vitamin A Deficiency / Vitamin A Deficiency <2017_cause_vitamin_a_deficiency>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Low Birth Weight and Short Gestation <2017_risk_lbwsg>`
 
-:ref:`Dietary Iron Deficiency / Iron Deficiency <2017_cause_iron_deficiency>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+PAF-of-1 Cause/Risk Pairs
++++++++++++++++++++++++++
 
-Additional GBD Risks
-++++++++++++++++++++
+* :ref:`Vitamin A Deficiency / Vitamin A Deficiency <2017_cause_vitamin_a_deficiency>`
 
-:ref:`Low Birth Weight and Short Gestation <2017_risk_lbwsg>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* :ref:`Dietary Iron Deficiency / Iron Deficiency <2017_cause_iron_deficiency>`
 
 Risk-Outcome Relationships
```

Comment:
Was wondering if this document is where the 'Concept model diagram' would go? And if so, where? The diagram of the treatment options, the risk factors, the causes, and desired outputs (morbidity, mortality)?

Reply:
Yep, concept model diagram was added in a previous commit. Here's the current version: https://vivarium-research.readthedocs.io/en/latest/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.html#concept-model-diagram